README.md

```diff
@@ -75,13 +76,16 @@ To run HospitalRun with Docker please do the following:
 - Build the HospitalRun image with `docker build -t hospitalrun-frontend .`
 - Execute `docker run -it --name couchdb -d couchdb` to create the couchdb container.
 - Execute `docker run -it --name hospitalrun-frontend -p 4200:4200 --link couchdb:couchdb -d hospitalrun-frontend` to create the HospitalRun container.
+or instead,
```

Comment:
The way this is formatted it isn't immediately clear that it is a different way to accomplish Running with Docker.  I suggest breaking the Running With Docker into two sub sections, eg:
##Running With Docker
###Running With Docker Engine
Existing information here
###Running with Docker Compose
Your new info here